docs/docs/tutorials/creating-cds-dashboard.md

```diff
@@ -3,45 +3,44 @@ id: creating-cds-dashboard
 title: Create and Deploy a dashboard
 ---
 
-# Create a dashboard with `Panel` and deploy it with `CDS Dashboards`
+# Create and deploy a dashboard with `Panel` and `CDS Dashboards`
```

Comment:
This main header is replaced by the docusaurus `title` above on the page tiles. 
![image](https://user-images.githubusercontent.com/15659110/176920045-89e67244-1792-4085-86bd-147fc9ebbc1c.png)

I'm happy with this title, but I was thinking you wanted to make it more descriptive?

Reply:
I still needed to come back and add a description